fbw-a32nx/src/wasm/fadec_a320/src/FadecGauge.h

```diff
@@ -200,6 +238,20 @@ class FadecGauge {
     }
   }
 
+  void simConnectProcessEvent(const SIMCONNECT_RECV_EVENT* event) {
```

Comment:
Should we just remove the method if it has no use?

Reply:
Well I wasnt sure if we want to do anything there in the future maybe. But yeah not really necessary to keep around till then